server/src/controllers/Chapter/resolver.ts

```diff
@@ -36,6 +36,9 @@ export class ChapterResolver {
     return await prisma.chapters.findMany({
       include: {
         events: {
+          where: {
+            AND: [{ canceled: false }, { ends_at: { gt: new Date() } }],
+          },
           include: {
```

Comment:
```suggestion
          orderBy: { start_at: 'asc' },
          include: {
```
Since we're here, might as well order them.